code/game/gamemodes/cult/runes/manifest_ghost.dm

```diff
@@ -45,21 +52,16 @@
 	log_and_message_admins("used a manifest rune.")
 
 	// The cultist doesn't have to stand on the rune, but they will continually take damage for as long as they have a summoned ghost
-	var/can_manifest = TRUE
-	while(user?.stat == CONSCIOUS && user.client && can_manifest)
-		can_manifest = user.species.take_manifest_ghost_damage(user)
+	while(user?.stat == CONSCIOUS && user.client)
+		user.take_organ_damage(1, 0)
 		sleep(30)
 	apparition_check()
```

Comment:
uuh. does this just kill the user now...? There's no other way to exit this loop.